src/main/groovy/ca/stellardrift/gitpatcher/task/patch/ApplyPatchesTask.groovy

```diff
@@ -39,6 +43,12 @@ import org.gradle.api.tasks.UntrackedTask
 
 @UntrackedTask(because = "State is tracked by git")
 abstract class ApplyPatchesTask extends PatchTask {
+    private GitPatcherExtension ext
```

Comment:
beyond just having a ListProperty for the extra arguments, maybe for ad-hoc customization we should have an `@Option`-annotated string property that is used to set the `convention` value of the extra arguments property, so extra args can be specified in the gradle invocation?

Reply:
It is techinally possible but I don't know what is the use case of it, please give me an example to understand it.